src/components/App.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div id="app">
+    <template v-if="$store.state.user">
+      <h2>You are logged in!</h2>
+      <div class="logout">
+        <button @click="logout">Logout</button>
+      </div>
+      <router-view></router-view>
+    </template>
+    <template v-else>
+      <div class="login">
+        <Auth />
```

Comment:
I think this line somehow goes and gets the login interface but I don't knwo how that works

My Vue knowledge is very shaky

Reply:
I think you answer your question in your comment below.

`Auth` is imported on L19.

Because it's a Vue Component, we can use it like an HTML tag inside the template.